appside/model/dbt_worksheet_model.js

```diff
@@ -56,6 +64,25 @@ class DbtWorksheetModelFwd extends Model {
         ]);
     }
 
+    /**
+     * Build likert frames for a DBT worksheet model as pre or post measurement frame.
+     */
+    build_pre_post_measurement_frames() {
+        let pre_post = {};
+
+        pre_post.title = LIKERT_TITLE;
+        pre_post.template = LIKERT_FRAME_TEMPLATE;
+        pre_post.instructions = LIKERT_INSTRUCTIONS;
+        
+        let pre_post_questions = [];
+        pre_post_questions.push(['Question 1', 2]);
```

Comment:
we know what questions these are now, so we should replace the placeholder strings with the actual questions. They can go in constants.js, with `SDERS` in the names.

(See email thread from Nov 15: "Feedback requested: DERS-S versus S-DERS")

The doc linked from the email also has specific instructions for our version of S-DERS, including the descriptions of each number in the scale.

Reply:
fixed :)
user_input still returns [question text, 1-5]